### PR TITLE
chore: Convert help command handling to typescript

### PR DESCRIPTION
**Pull request by lili2311**
- [x] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?

Convert the `help` command handler to TypeScript

#### Any background context you want to provide?
Slowly converting to TS 🐌

### CODE REVIEW

File: src/cli/commands/test.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import {SEVERITIES, WIZARD_SUPPORTED_PMS} from '../../lib/snyk-test/common';
 import * as Debug from 'debug';
 import {TestOptions} from '../../lib/types';
 import {isLocalFolder} from '../../lib/detect';
-import { MethodArgs, ArgsOptions } from '../args';
+import { MethodArgs } from '../args';
```
Comment (lili2311) [inline src/cli/commands/test.ts]: unused import

File: src/cli/commands/help.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import * as fs from 'then-fs';
+import * as path from 'path';
+import * as Debug from 'debug';
+const debug = Debug('snyk');
```
Comment (kyegupov) [inline src/cli/commands/help.ts]: I usually use "debugModule". I'd like to write down an unified style guide somewhere :)

File: src/cli/commands/help.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  // cleanse the filename to only contain letters
+  // aka: /\W/g but figured this was easier to read
+  item = item.replace(/[^a-z-]/gi, '');
+
+  const filename = path.resolve(__dirname, '../../../help', item + '.txt');
+  try {
+    await fs.readFile(filename, 'utf8');
+  } catch (error) {
+    debug(error);
+    return `'${item}' help can\'t be found at location: ${filename}`;
```
Comment (kyegupov) [inline src/cli/commands/help.ts]: no need to backslash a single quote?

File: src/cli/commands/help.ts (hunk 2 above)
Comment (lili2311) [reply src/cli/commands/help.ts]: Styleguide would be good, but the CLI mostly uses `debug`

**Pull request merged**